backend/__init__.py

```diff
@@ -0,0 +1,23 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+# init SQLAlchemy so we can use it later in our models
+db = SQLAlchemy()
+@app.route("/")
+def create_app():
+    app = Flask(__name__)
+
+    app.config['SECRET_KEY'] = 'secret-key-goes-here'
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
```

Comment:
Merge work with Jordan he has setup a database connection with sqlalchemy

Reply:
Also check out this repo for more ideas https://github.com/Dave-mash/Questioner-api-v2

Reply:
use postgresql database and create a config.py file with  development, production, and testing methods

# Production database
class Production(Config):
    SQLALCHEMY_DATABASE_URI = ''
    SQLALCHEMY_TRACK_MODIFICATIONS = False
    ENV = "production"
    DEBUG = False